src/spi.h

```diff
@@ -59,6 +59,7 @@ typedef struct FLASHER_SPI_CONFIGURATION_STRUCT
 	unsigned int uiUnit;              /**< @brief SPI unit number. */
 	unsigned int uiChipSelect;        /**< @brief The chip select of the SPI unit. */
 	unsigned long ulInitialSpeedKhz;  /**< @brief The initial speed during identification. */
+	unsigned long ulManualSpeedKhz;   /**< @brief The manually set speed. */
```

Comment:
This changes the flasher's API and we need to indicate that in flasher_interface.h:
#define FLASHER_INTERFACE_VERSION 0x00030000
This indicates version 3.0. We need to change it to either 3.1 (0x00030001) or 4.0, I'm not sure which.
If you rebuild the flasher, the change should automatically go into flasher.lua.